catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -147,8 +147,8 @@
   @Argument(
     name = "File path or Directory path",
     description =
-        "File path to a record or a directory of files to be ingested. Paths are absolute and must be in quotes."
-            + " This command can only detect roughly 2 billion records in one folder. Individual operating system limits might also apply.",
+        "File path to a file or a directory of files to be ingested. Paths can be absolute or relative to installation directory."
+            + " This command can only detect roughly 2 billion files in one folder. Individual operating system limits might also apply.",
```

Comment:
✏️ "...files in one directory. Individual..." 
`folder` should be reserved for gui references.

Reply:
fixed.